org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/preprocess/PreprocessTest.java

```diff
@@ -92,7 +92,7 @@ public void testMalcolmPreprocess() throws Exception {
 
 		final ScanRequest<?> req = new ScanRequest<IROI>();
 		req.setCompoundModel(new CompoundModel(new StepModel("fred", 0, 9, 1)));
-		req.setMonitorNames(Arrays.asList("monitor"));
+		req.setMonitorNamesPerPoint(Arrays.asList("monitor"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "monitor" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)